gammapy/data/observations.py

```diff
@@ -240,35 +241,31 @@ def observation_dead_time_fraction(self):
         """
         return 1 - self.obs_info["DEADC"]
 
+    @lazyproperty
+    def fixed_pointing_info(self):
+        """Fixed pointing info for this observation (`FixedPointingInfo`)."""
+        return FixedPointingInfo(self.events.table.meta)
```

Comment:
For simulations the event meta will never be there from the beginning. Doesn't it work to just change to `FixedPointingInfo(self.obs_info)` for now and then later improve the handling of the "in-memory" meta and `events.table.meta`, that we planned to do anyway?

Reply:
That didn't work since the obs info table used for obs info in case I going through datastore does not contain the mjdref values. I think these are in the header of the obs table but are not added to the o s info at the moment.

Reply:
I think adding the `MJDREF` info from the observation table header is probably fine for now? It should be a mandatory keyword...

Reply:
And for the simulation case one could adapt the `Observation._get_obs_info`...

Reply:
Just to add: I think This will not be any final solution, but it will probably help to identify and improve the internal data model.

Reply:
done